app/controllers/expenses_controller.rb

```diff
@@ -0,0 +1,38 @@
+class ExpensesController < ApplicationController
+  
+  def create
+    begin
+      expense = Expense.create(expense_params)
+      render status: :created, json: expense.to_json
+    rescue
+      render status: 400
+    end
+  end
+
+  def destroy
+    begin
+      params.require(:id)
+      Expense.destroy_by(id: params[:id])
+      render status: :ok
+    rescue
+      render status: 400
+    end
+  end
+
+  def update
+    begin
+      id = params[:id]  
+      expense = Expense.find_by(id: id)    
+      expense.update(expense_params)
+    rescue
+      render status: 400
+    end
+  end
+
+  private
+
+  def expense_params
+    params.require(:expense).require(:name, :amount, :due_date, :debtor_id, :hideout_id, :comments)
```

Comment:
I mentioned this on vik's PR but we decided to remove this function since it requires things like comments and due_date and debtor_id and stuff. Since we don't have so many functions in the first place and all other controller functions use individualized `params.require` calls, just create individual `params.require` calls in each function here. In other words, remove this helper.

Reply:
Alright I will remove it for now for consistency. Although it may be better in the future to refactor the other controllers to use a private params method like this since this is the pattern recommended in the rails docs.


![image](https://github.com/tryhideout/web/assets/92739966/0392a399-40c8-4074-8a07-eeff95ede49d)